src/pages/docs/supporting-projects/emulsify-core/advanced-usage/3rd-party-libraries.md

```diff
@@ -4,7 +4,11 @@ pageTitle: 'Emulsify Drupal: Adding 3rd-party libraries'
 description: How to add libraries like jQuery to your project.
 ---
 
-## Global Libraries \(e.g. jQuery\)
```

Comment:
We need to add this feature to Emulsify Core. Hiding these examples until this feature is publish in Emulsify Core 2.4.